lib/RobloxRenderer.spec.lua

```diff
@@ -73,6 +74,77 @@ return function()
 	end)
 
 	describe("updateHostNode", function()
+		it("should update node props and children", function()
+			local parent = Instance.new("Folder")
+			local key = "updateHostNodeTest"
+			local firstValue = "foo"
+			local newValue = "bar"
+
+			local _, defaultStringValue = getDefaultPropertyValue("StringValue", "Value")
+
+			local element = createElement("StringValue", {
+				Value = firstValue
+			}, {
+				ChildA = createElement("IntValue", {
+					Value = 1
+				}),
+				ChildB = createElement("BoolValue", {
+					Value = false,
```

Comment:
We should set this value to `true` so that when we reconcile over the top, we're actually changing the value!

Reply:
Oops, you're right!